cmd/zt_generic_traverser_test.go

```diff
@@ -170,6 +172,75 @@ func (s *genericTraverserSuite) TestS3GetProperties(c *chk.C) {
 	c.Assert(seenContentType, chk.Equals, true)
 }
 
+func (s *genericTraverserSuite) TestGCPGetProperties(c *chk.C) {
+	skipIfS3Disabled(c)
+	client, err := createGCPClientWithGCSSDK()
+
+	if err != nil {
+		c.Skip("GCP-based tests will not be run as no credentials were supplied.")
+		return
+	}
+
+	headers := storage.ObjectAttrsToUpdate{
+		ContentType:        "text/html",
+		ContentEncoding:    "gzip",
+		ContentLanguage:    "en",
+		ContentDisposition: "inline",
+		CacheControl:       "no-cache",
+	}
+
+	bucketName := generateBucketName()
+	objectName := generateObjectName()
+	bkt := client.Bucket(bucketName)
+	err = bkt.Create(context.Background(), os.Getenv("GOOGLE_CLOUD_PROJECT"), &storage.BucketAttrs{})
```

Comment:
>os.Getenv("GOOGLE_CLOUD_PROJECT") [](start = 40, length = 33)

use lifecycle manager instead to fetch the env var

Reply:
This is for the testcase and not built into binary. We've preferred fetching directly from Env in testcases.